.github/workflows/push.yml

```diff
@@ -78,5 +81,6 @@ jobs:
         with:
           runTestsParameters: >-
             -d zend_extension=opcache.so
+            -d opcache.enable_cli=1
```

Comment:
```
 ========DIFF========
002+ ERROR: No starting lines
     Done
========DONE========
FAIL FPM: bug76601 children should not ignore signals during concurrent reloads [sapi/fpm/tests/bug76601-reload-child-signals.phpt] 
```

JIT seems to have an issue on MacOS

Reply:
cannot reproduce locally but I run under Monterey and ARM arch in my case.

Reply:
thanks, pushed again, maybe some random CI failure